lesson_07/objects/objects_app/src/main/java/com/codedifferently/lesson7/Jordan_Eldridge/Animal.java

```diff
@@ -0,0 +1,121 @@
+package com.codedifferently.lesson7.Jordan_Eldridge;
+
+public class Animal {
```

Comment:
Missing a collection and conditions.

Reply:
I did not see that in the GitHub requirements is that something mentioned during the lecture ?

Reply:
Boo, I totally forgot to add that. Nevermind, please ignore.

Reply:
you the man !

Reply:
this assignment is good though alot of them are struggling with tests!